mysql-test/suite/innodb/t/percona_changed_page_bmp_1364315.test

```diff
@@ -6,9 +6,7 @@
 
 --let $TMPDATADIR= $MYSQLTEST_VARDIR/tmpdatadir_1364315
 
---exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
---shutdown_server 10
---source include/wait_until_disconnected.inc
+--source include/shutdown_mysqld.inc
 --mkdir $TMPDATADIR
 --enable_reconnect
 --echo 1st restart
```

Comment:
`--source include/start_mysqld.inc` with parameters instead of `--source include/wait_until_connected_again.inc` here?